programs/vest_api/src/vest_instruction.rs

```diff
@@ -88,25 +88,25 @@ fn initialize_account(
 }
 
 pub fn create_account(
-    terminator_pubkey: &Pubkey,
+    custodian_pubkey: &Pubkey,
     contract_pubkey: &Pubkey,
-    payee_pubkey: &Pubkey,
+    withdrawer_pubkey: &Pubkey,
     start_date: Date<Utc>,
     date_pubkey: &Pubkey,
     lamports: u64,
 ) -> Vec<Instruction> {
     let space = serialized_size(&VestState::default()).unwrap();
     vec![
         system_instruction::create_account(
-            &terminator_pubkey,
+            &custodian_pubkey,
             contract_pubkey,
             lamports,
             space,
             &id(),
         ),
         initialize_account(
-            terminator_pubkey,
-            payee_pubkey,
+            custodian_pubkey,
```

Comment:
why overload custodian here?  custodian funds, but might not be the same entity/key as the guy who'd terminate

Reply:
good point, I'll update